test/cases/regression_tests

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
+"""
+Tests the osbuild-image-test by running it against two fake DBs. This test must
+run on Fedora35 x86_64
+"""
+
+import subprocess
+import configparser
+import platform
+import sys
+from subprocess import CalledProcessError
+
+OS_RELEASE_PATH = "/etc/os-release"
+
+config = configparser.ConfigParser()
+with open(OS_RELEASE_PATH, 'r', encoding='utf-8') as f:
+    config_string = '[DEFAULT]\n' + f.read()
+config.read_string(config_string)
+distro = f"{config.get('DEFAULT', 'ID')}-{config.get('DEFAULT', 'VERSION_ID')}"
+distro = distro.replace('"', '')
+distro = distro.replace('.', '')
+
+print(f"Testing the osbuild-image-test for arch {platform.machine()} and "
+        f"distribution {distro} on folder test/data/db-success "
+        "this test must succeed", flush=True)
+
+try:
+    subprocess.run(["tools/osbuild-image-test",
+                    f"--arch={platform.machine()}",
+                    f"--distro={distro}",
+                    "--db-path=test/data/db-success"],
+                    check=True)
+except CalledProcessError:
+    sys.exit(1)
+else:
+    print("SUCCESS")
+
+print(f"Testing the osbuild-image-test for arch {platform.machine()} and "
+        f"distribution {distro} on folder test/data/db-failure "
+        "this test must fail", flush=True)
+
+try:
+    subprocess.run(["tools/osbuild-image-test",
+                    f"--arch={platform.machine()}",
+                    f"--distro={distro}",
+                    "--db-path=test/data/db-success"],
+                    check=True)
+except CalledProcessError:
+    sys.exit(1)
+else:
+    print("SUCCESS")
```

Comment:
Should this be run on `db-failure` and the condition flipped?

Reply:
Ahah! Thanks, yes clearly that's a half copy/paste issue here.